src/main/java/bio/terra/testrunner/common/utils/KubernetesClientUtils.java

```diff
@@ -371,6 +427,17 @@ private static Container createContainerService(GoogleCredentials credentials)
     return list.getItems();
   }
 
+  /**
+   * Get the Kubernetes ConfigMap identified by the name argument.
+   *
+   * @param name the name of the Kubernetes ConfigMap
+   * @return a V1ConfigMap object
+   */
+  public static V1ConfigMap getComponentVersionConfigMap(String name) throws ApiException {
```

Comment:
Where is this called?

Reply:
Forgot to remove this method since it has been refactored into this method: `importComponentVersions`. Removed it now.